ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -479,6 +479,14 @@ tests:
   steps:
     cluster_profile: ibmcloud
     workflow: openshift-e2e-ibmcloud
+- as: e2e-metal-upi
+  optional: true
+  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|terraform/providers)/(alibabacloud|aws|azure|azurestack|baremetal|gcp|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

We don't want this job to run very often. I think
```suggestion
  run_if_changed: metal
```
would make it run for changes to baremetal or upi/metal, which seems good to me.